cmd/aro/update_ocp_versions.go

```diff
@@ -31,11 +31,19 @@ func getLatestOCPVersions(ctx context.Context, log *logrus.Entry) ([]api.OpenShi
 	ocpVersions := []api.OpenShiftVersion{}
 
 	for _, vers := range version.HiveInstallStreams {
+		installerPullSpec := fmt.Sprintf("%s/aro-installer:%s", dstRepo, vers.Version.MinorVersion())
+		digest, ok := version.InstallerImageDigest[vers.Version.MinorVersion()]
+		if !ok {
```

Comment:
Should we leave this out? To me it looks like we're still quietly supporting non-tagged when we want to use SHA always.

Reply:
I think that's fine.  Let's just error out if the SHAs are not present.  I don't think anyone is using this for hive and if they are we can get docs updated.

Reply:
Updated to return an error directly if one doesn't exist.